Source/Scene/Model.js

```diff
@@ -869,7 +944,13 @@ define([
             if (buffers.hasOwnProperty(name)) {
                 var buffer = buffers[name];
 
-                if (buffer.type === 'arraybuffer') {
+// TODO: Best semantic for this?
```

Comment:
Is this TODO still valid?  What exactly are you unsure about, using the name `self`?  In general, I would advise against using it because it already has a meaning [elsewhere](https://developer.mozilla.org/en-US/docs/Web/API/Window/self). @shunter any thoughts?